src/types/reduxActions.ts

```diff
@@ -172,7 +173,7 @@ export type Action =
   | { type: 'UI/WALLETS/UPSERT_WALLETS'; data: { wallets: EdgeCurrencyWallet[] } }
   | { type: 'UPDATE_SORTED_WALLET_LIST'; data: WalletListItem[] }
   | { type: 'UPDATE_SWAP_QUOTE'; data: GuiSwapInfo }
-  | { type: 'UPDATE_SHOW_PASSWORD_RECOVERY_REMINDER_MODAL'; data: number }
+  | { type: 'UPDATE_SHOW_PASSWORD_RECOVERY_REMINDER_MODAL'; data: keyof typeof PASSWORD_RECOVERY_REMINDERS_SHOWN }
```

Comment:
Maybe we could export `type PasswordReminderTime =   keyof typeof PASSWORD_RECOVERY_REMINDERS_SHOWN` or something like that, so we don't have to import the actual table, just the type information.